appeals/web/src/server/appeals/appeal-details/appellant-case/appellant-case.service.js

```diff
@@ -0,0 +1,70 @@
+import { get, patch } from '../../../lib/request.js';
+
+/**
+ * @typedef {import('./appellant-case.types.js').SingleAppellantCaseResponse} AppellantCaseResponse
+ */
+
+/**
+ * @param {number} appealId
+ * @param {number} appellantCaseId
+ */
+export function getAppellantCaseFromAppealId(appealId, appellantCaseId) {
+	return get(`appeals/${appealId}/appellant-cases/${appellantCaseId}`);
+}
+
+/**
+ * @typedef {Object} DayMonthYear
+ * @property {number} day
+ * @property {number} month
+ * @property {number} year
+ */
+
+/**
+ * @typedef {Object} AppellantCaseReviewOutcome
+ * @property {string} validationOutcome
+ * @property {number[]} [invalidReasons]
+ * @property {number[]} [incompleteReasons]
+ * @property {string} [otherNotValidReasons] - free text field for "other" (shared by both invalid and incomplete outcomes)
+ * @property {DayMonthYear} [updatedDueDate]
+ */
+
+/**
+ * @param {number} appealId
+ * @param {number} appellantCaseId
+ * @param {AppellantCaseReviewOutcome} reviewOutcome
+ * @returns {Promise<AppellantCaseResponse>}
+ */
+export async function setReviewOutcomeForAppellantCase(appealId, appellantCaseId, reviewOutcome) {
+	return patch(`appeals/${appealId}/appellant-cases/${appellantCaseId}`, {
+		json: reviewOutcome
+	});
+}
+
+/**
+ * @returns {Promise<import('appeals/api/src/database/schema.js').AppellantCaseInvalidReason[]>}
+ */
+export async function getAppellantCaseInvalidReasons() {
+	return get(`appeals/appellant-case-invalid-reasons`);
+}
+
+/**
+ * @returns {Promise<import('appeals/api/src/database/schema.js').AppellantCaseIncompleteReason[]>}
+ */
+export async function getAppellantCaseIncompleteReasons() {
+	return get(`appeals/appellant-case-incomplete-reasons`);
+}
+
+/**
+ * @param {import('./appellant-case.service.js').AppellantCaseReviewOutcome} appellantCaseReviewOutcome
+ * @returns {Promise<import('appeals/api/src/database/schema.js').AppellantCaseInvalidReason[]|import('appeals/api/src/database/schema.js').AppellantCaseIncompleteReason[]>}
+ */
+export async function getAppellantCaseNotValidReasonsForOutcome(appellantCaseReviewOutcome) {
+	switch (appellantCaseReviewOutcome.validationOutcome) {
+		case 'invalid':
```

Comment:
We've got these strings dotted around, seems like a good time to put in a constant file to reference by name.

Reply:
Moved these into `appeals/web/src/server/appeals/appeal.constants.js`